### DISCUSSION TITLE
Asked to Update then Lost all Templates

### DISCUSSION THREAD

**Discussion by gamesinplay** [coddingtonbear/obsidian-web#180] [Q&A] [answered]
Somewhere, either in Obsidian or the Extension asked me to update obsidian-web. When I did I lost all of my templates. 

I worked them all out a year or more ago and don't recall how to do most template language now.  So, before I start over...

is there somewhere on my Windows PC I could look for a record of the templates? Thanks in advance.

**Comment by coddingtonbear** [comment, marked as answer]
I'm afraid they're all handled via Google Chrome's syncing machinery; so I'm not sure if they're on disk anywhere accessible at all.  I can say that if you go to the options page and press the "export" button in the upper-right corner, the raw contents of the relevant settings will be included in there under the `sync` key's `presets` key, and you can tell what your templates looked like at a glance.

That said, I don't believe we've changed the data format for templates (`presets` in the settings file) ever, though; so it's pretty unlikely that the problem is just some kind of bad data migration.  I'm afraid the likely story is that your settings are just lost forever for some reason other than the upgrade -- maybe your particular browser deletes unused extensions' settings after a time or something like that?

**Reply by gamesinplay** [reply]
Thank you.
